plugins/justify/plugin.js

```diff
@@ -178,8 +188,17 @@
 
 		refresh: function( editor, path ) {
 			var firstBlock = path.block || path.blockLimit;
-
-			this.setState( firstBlock.getName() != 'body' && getAlignment( firstBlock, this.editor.config.useComputedState ) == this.value ? CKEDITOR.TRISTATE_ON : CKEDITOR.TRISTATE_OFF );
+			var name = firstBlock.getName();
+
+			// Check if we can give to element proper class, style or current path is body (#455).
+			// IE8, after focus and refresh return path with 'body'
+			if ( ( this.cssClassName && editor.activeFilter.check( name + '(' + this.cssClassName + ')' ) ) ||
+				editor.activeFilter.check( name + '{text-align}' ) ||
+				name == editor.editable().getName() ) {
```

Comment:
Ok this example showed me what you have on your mind.
Generally 3rd condition was left to preserve possibility to justify in ENTER_BR mode.

Right now I restrict this option to BR mode only ;)
Because of that I have to modify one test condition.
To have coherent visualisation between different editor types and enter modes, button will be turned off when selection will be in `editor.editable()`.